source/tests/unit/ni_fake_non_ivi_service_tests.cpp

```diff
@@ -1154,22 +1158,20 @@ TEST_F(NiFakeNonIviServiceTests, SetColors_PassesColorsArrayToLibrary)
   EXPECT_EQ(kDriverSuccess, response.status());
 }
 
-struct NiFakeNonIviServiceEnabledToggleTests : public NiFakeNonIviServiceTests {
-  NiFakeNonIviServiceEnabledToggleTests()
-      : NiFakeNonIviServiceTests(
-            nidevice_grpc::FeatureToggles({{"nifake_non_ivi", true}}))
-  {
-  }
-};
-
-TEST_F(NiFakeNonIviServiceEnabledToggleTests, UnreleasedServiceWithToggleEnabled_IsEnabled_ReturnsTrue)
+TEST_F(NiFakeNonIviServiceTests, UnreleasedServiceWithToggleEnabled_IsEnabled_ReturnsTrue)
```

Comment:
Note: rewriting these tests, since `is_enabled` is now used external to the service (in the registrar).